src/main/kotlin/io/arct/arctlib/utils/Cooldown.kt

```diff
@@ -0,0 +1,63 @@
+package io.arct.arctlib.utils
+
+import java.util.*
+
+/**
+ * Simulate a cooldown given a UUID and duration
+ *
+ * @param id The UUID assigned to this cooldown
+ * @param duration The duration of the cooldown in seconds
+ * @constructor Starts the cooldown
+ */
+class Cooldown(id: UUID, val duration: Int) {
```

Comment:
This is kind of a weird pattern, because you create a new object (`Cooldown`) to create a cooldown. Maybe this should be an object instead that has a function that lets you create a cooldown instead?

Reply:
wdym